<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Nop Commerce" duration-ms="25102" started-at="2020-10-11T13:14:19Z" finished-at="2020-10-11T13:14:44Z">
    <groups>
    </groups>
    <test name="Run on " duration-ms="25102" started-at="2020-10-11T13:14:19Z" finished-at="2020-10-11T13:14:44Z">
      <class name="com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator">
        <test-method status="PASS" signature="beforeTest(java.lang.String)[pri:0, instance:com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator@4b6995df]" name="beforeTest" is-config="true" duration-ms="8700" started-at="2020-10-11T20:14:19Z" finished-at="2020-10-11T20:14:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="TC_01_RegisterToSystem()[pri:0, instance:com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator@4b6995df]" name="TC_01_RegisterToSystem" duration-ms="7880" started-at="2020-10-11T20:14:27Z" finished-at="2020-10-11T20:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_01_RegisterToSystem -->
        <test-method status="PASS" signature="TC_02_LoginToSystem()[pri:0, instance:com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator@4b6995df]" name="TC_02_LoginToSystem" duration-ms="3966" started-at="2020-10-11T20:14:35Z" finished-at="2020-10-11T20:14:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_02_LoginToSystem -->
        <test-method status="FAIL" signature="TC_03_Dynamic_less()[pri:0, instance:com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator@4b6995df]" name="TC_03_Dynamic_less" duration-ms="2880" started-at="2020-10-11T20:14:39Z" description="Only use incase less page" finished-at="2020-10-11T20:14:42Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[pageObjects_nopcommerce.FooterMyAccountPageObject cannot be cast to pageObjects_nopcommerce.FooterNewProductsPageObject]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: pageObjects_nopcommerce.FooterMyAccountPageObject cannot be cast to pageObjects_nopcommerce.FooterNewProductsPageObject
	at com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator.TC_03_Dynamic_less(Login_08_RegisterAndLogin_Dynamic_Locator.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_03_Dynamic_less -->
        <test-method status="FAIL" signature="TC_04_Dynamic_More()[pri:0, instance:com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator@4b6995df]" name="TC_04_Dynamic_More" duration-ms="1" started-at="2020-10-11T20:14:42Z" finished-at="2020-10-11T20:14:42Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator.TC_04_Dynamic_More(Login_08_RegisterAndLogin_Dynamic_Locator.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_04_Dynamic_More -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator@4b6995df]" name="afterTest" is-config="true" duration-ms="1660" started-at="2020-10-11T20:14:42Z" finished-at="2020-10-11T20:14:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.login.nopcommerce_login.Login_08_RegisterAndLogin_Dynamic_Locator -->
    </test> <!-- Run on  -->
  </suite> <!-- Nop Commerce -->
</testng-results>
